File: BE/capsule-service/src/main/java/com/timecapsule/capsuleservice/api/request/MemoryRegistReq.java
package com.timecapsule.capsuleservice.api.request;

import lombok.Getter;

import java.time.LocalDate;

@Getter
public class MemoryRegistReq {
    private int memberId;
    private int capsuleId;
    private String title;
    private String content;
    private LocalDate openDate;
}

File: FE/AboutTimeCapsule/app/src/main/java/com/aboutcapsule/android/data/memory/MemoryRegistReq.kt
package com.aboutcapsule.android.data.memory

import com.google.gson.annotations.SerializedName
import java.time.LocalDate

data class MemoryRegistReq(
    val memberId: Int,
    val capsuleId: Int,
    val title: String,
    val content: String,
    val openDate: String,
)
